src/material/slider/slider.scss

```diff
@@ -1,123 +1,139 @@
+
+    display: inline-block;
+    box-sizing: border-box;
+    outline: none;
+    vertical-align: middle;
+    margin: {
+      left: $mat-slider-horizontal-margin;
+      right: $mat-slider-horizontal-margin;
     }
-  }
 
-  .mdc-slider__thumb,
-  .mdc-slider__track--active_fill {
-    transition-duration: 0ms;
-  }
-  &.mat-mdc-slider-with-animation {
-    .mdc-slider__thumb,
-    .mdc-slider__track--active_fill {
-      transition-duration: 80ms;
+    // Unset the default "width" property from the MDC slider class. We don't want
+    // the slider to automatically expand horizontally for backwards compatibility.
+    width: auto;
+    min-width: $mat-slider-min-size - (2 * $mat-slider-horizontal-margin);
+    -webkit-tap-highlight-color: transparent;
+
+    .mdc-slider__input {
+      // It's common for apps to globally set `box-sizing: border-box` which messes up our
+      // measurements. Explicitly set `content-box` to avoid issues like #26246.
+      box-sizing: content-box;
+      pointer-events: auto;
+
+      &.mat-mdc-slider-input-no-pointer-events {
+        pointer-events: none;
+      }
+
+      &.mat-slider__right-input {
+        left: auto;
+        right: 0;
+      }
     }
-  }
 
-  // We need to repeat these styles to override discrete slider styling.
-  &.mdc-slider--discrete {
     .mdc-slider__thumb,
     .mdc-slider__track--active_fill {
       transition-duration: 0ms;
     }
-  }
-  &.mat-mdc-slider-with-animation {
-    .mdc-slider__thumb,
-    .mdc-slider__track--active_fill {
-      transition-duration: 80ms;
+    &.mat-mdc-slider-with-animation {
+      .mdc-slider__thumb,
+      .mdc-slider__track--active_fill {
+        transition-duration: 80ms;
+      }
     }
-  }
 
-  .mdc-slider__track,
-  .mdc-slider__thumb {
-    pointer-events: none;
-  }
+    // We need to repeat these styles to override discrete slider styling.
+    &.mdc-slider--discrete {
+      .mdc-slider__thumb,
+      .mdc-slider__track--active_fill {
+        transition-duration: 0ms;
+      }
+    }
+    &.mat-mdc-slider-with-animation {
+      .mdc-slider__thumb,
+      .mdc-slider__track--active_fill {
+        transition-duration: 80ms;
+      }
+    }
 
-  .mdc-slider__value-indicator {
-    // There's no token to control this opacity so we have to do it ourselves.
-    opacity: var(--mat-mdc-slider-value-indicator-opacity, 1);
-  }
+    .mdc-slider__track,
+    .mdc-slider__thumb {
+      pointer-events: none;
+    }
 
-  // The `.mat-ripple` wrapper here is redundant, but we need it to
-  // increase the specificity due to how some styles are setup in g3.
-  .mat-ripple {
-    // These ripple colors are custom so we have to introduce our own tokens.
-    .mat-ripple-element {
-      background-color: var(--mat-mdc-slider-ripple-color, transparent);
+    .mdc-slider__value-indicator {
+      // There's no token to control this opacity so we have to do it ourselves.
+      opacity: var(--mat-mdc-slider-value-indicator-opacity, 1);
```

Comment:
Shouldn't we be using the `token-utils.create-token-slot` to declare the slots here?

Reply:
This isn't a token that we can pass into MDC, but gets defined [here](https://github.com/material-components/material-components-web/blob/master/packages/mdc-slider/_slider-theme.scss#L47). We could add this as a token on our end but I'm not too certain if people would want to change the opacity level. 

@crisbeto do you think we should add this custom token or leave this as is? I think either option makes sense.

Reply:
If we don't intend for it to change at all, we should just hardcode it to the value. If we think that it may need to change in the future (e.g. for M3), we can put it in the unthemable tokens.